content/v2.0/reference/key-concepts.md

```diff
@@ -0,0 +1,184 @@
+
+### Sample data
+
+The sample data below shows a number of bees counted by two scientists (`anderson` and `mullen`) in two locations (`1` and `2`) from 12 AM to 6 AM on August 18, 2019. The sample data is stored in a bucket `my_bucket` and retained for the duration of the retention policy specified in the [bucket](#bucket).
+
+**_Tip:_** Hover over purple terms to get acquainted with InfluxDB terminology and layout.
+
+bucket:  `my_bucket`
+
+| _time | _measurement| <span class="tooltip" data-tooltip-text="Field key">_field</span>|<span class="tooltip" data-tooltip-text="Field value">_value</span>|<span class="tooltip" data-tooltip-text="Tag key">location</span>|<span class="tooltip" data-tooltip-text="Tag key">scientist</span>|
+|:-------------------  |:------------|:--|:---|:-------|:------|
+| 2019-08-18T00:00:00Z | census|bees |23 | 1  |anderson|
+| 2019-08-18T00:00:00Z | census|bees |30 | 1  |mullen  |
+| 2019-08-18T00:06:00Z | census|bees |28 |  2 |anderson|
+|                      |       |     |   |    |        |       |
+| <span class="tooltip" data-tooltip-text="Timestamp">2019-08-18T00:06:00Z</span>| <span class="tooltip" data-tooltip-text="measurement"> census</span>| <span class="tooltip" data-tooltip-text="Field key">ants</span>| <span class="tooltip" data-tooltip-text="Field value">3</span> | <span class="tooltip" data-tooltip-text="Tag value">2</span> |<span class="tooltip" data-tooltip-text="Tag value">mullen</span>|
+
+#### Timestamp
+
+All data stored in InfluxDB has a `_time` column that stores timestamps. Timestamps show the date and time in [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) UTC associated with data. Timestamp precision is important. When you search data within a specified time interval, make sure the timestamp precision you're searching matches the timestamp precision in your dataset.
+
+#### Measurement
+
+The  `_measurement` column shows the name of the measurement `census`. Measurement names are strings. A measurement acts as a container for tags, fields, and timestamps. Use a measurement name that describes your data. The name `census` tells us that the field values record the number of `bees` and `ants`. A single measurement can belong to different [buckets](#bucket).
+
+#### Fields
+
+A field includes a field key stored in (`_field`) and associated field value(s) stored in (`_value`).
+
+##### Field key
+
+The field keys `bees` and `ants` is a string that stores the name of the field.
+
+##### Field values
+
+The field values are your data; they can be strings, floats, integers, or Booleans. A field value always has an associated timestamp. The field values in the sample data show the number of `bees` at specified times: `23`, `30`, and `28` and the number of `ants` at a specified time: `3`.
+
+##### Field sets
+
+A field set is a collection of field key-value pairs. The sample data includes the following four field sets:
+
+- `bees = 23`
+- `bees = 30`
+- `bees = 28`
+- `ants = 3`
```

Comment:
A field set is a collection of field key-value pairs associated with a timestamp.

Field sets a generally only structured as sets in line protocol, but It's ok to call them out here. Where two different fields are recorded at the same time, there's actually 2 field sets in the sample. And I'd just format this as a code block rather than a list. Here's the actual line protocol.

```
census bees=23i,ants=30i 1566086400000000000
census bees=28i,ants=32i 1566086760000000000
       -----------------
           Field set
```